packages/core-bridge/scripts/build.js

```diff
@@ -65,9 +66,10 @@ if (unsupportedTargets.length) {
   process.exit(1);
 }
 const forceBuild = args['--force'];
+const buildRelease = args['--release'] || process.env.BUILD_CORE_RELEASE !== undefined;
```

Comment:
nit: `['y', 'yes', 't', 'true', '1'].includes((process.env.BUILD_CORE_RELEASE ?? '').toLowerCase())`

Reply:
Ehh, I'm not a fan TBH. If you don't want the env var set, just unset it. IMO best practice here is to have boolean-like env vars always default to false and any setting means true. That way there's no reason to ever set it explicitly to false. If you want to unset it, just unset the env var.

Reply:
I just feel like anyone setting `BUILD_CORE_RELEASE=0` would be surprised with the current behavior.

Reply:
Why would you ever do that, is my point. Just unset it, or never set it in the first place, since the default's false. It's a very common pattern IME